classes/class-query.php

```diff
@@ -216,6 +216,16 @@ public function query( $args ) {
 
 		$select = implode( ', ', $selects );
 
+		/**
+		 * Filters query WHERE statement as an alternative to filtering
+		 * the $query using the hook below.
+		 *
+		 * @param string $where  WHERE statement.
+		 *
+		 * @return string
+		 */
+		$where = apply_filters( 'wp_stream_db_query', $where );
```

Comment:
@kidunot89 I would expect the filter name to be different from `wp_stream_db_query`, which is already used below. How about `wp_stream_db_query_where` instead?

Reply:
:man_facepalming: my bad